deployment/helm/values.yaml

```diff
@@ -25,6 +25,11 @@ rbac:
   # enabled defines whether to apply rbac to Akri
   enabled: true
 
+# namespace is the Kubernetes namespace that will be created for storing
+# Akri related deployments
+namespace:
```

Comment:
why are introducing two levels here (namespace.name) rather than just one level (namespace)?

Reply:
Just to make more clear and follow the patterns from the other values (service_name.data), but I guess it could've be in a single level variable.